Gateway/Response/VaultDetailsHandler.php

```diff
@@ -55,9 +64,16 @@ public function handle(array $handlingSubject, array $response)
         /** @var PaymentDataObject $orderPayment */
         $orderPayment = SubjectReader::readPayment($handlingSubject);
         $additionalData = $orderPayment->getPayment()->getAdditionalInformation();
+
+        if ($this->adyenHelper->isCreditCardVaultEnabled()) {
+            $this->vaultHelper->saveRecurringDetails($orderPayment->getPayment(), $response['additionalData']);
+        }
+
+        /*
+         * Token enabler is only sent from the admin payment while we should not rely on such a checkbox
```

Comment:
@AlexandrosMor could you pls create a ticket to clean up the is_active_payment_token_enabler and start using the storePaymentMethod also in the admin if still applicable?

Reply:
The is_active_payment_token_enabler is checked by the magento in order to save the token. Magento is checking 
```
 const IS_ACTIVE_CODE = 'is_active_payment_token_enabler';
and is used in magento: 

$paymentToken->setIsVisible(
            (bool) (int) ($additionalInformation[VaultConfigProvider::IS_ACTIVE_CODE] ?? 0)
        );
        ```